[IMP] account_invoice_tax: better ux + protect analytic taxes

feg-adhoc opened the pull request:
1. Mostramos símbolo de la moneda (usamos field monetary)
2. Mejoramos para que en notas de crédito se vea importe positivo si impuesto es postivo
3. Agregamos protección para que no se usen impuestos con setup de analitico porque no está bien implementado y, según analizamos:
* tiene poco uso: solo primicia, gestionen, linos y londero
* no tiene tanto sentido contable
* es complejo de resolver, habŕia que ver si seguimos mostrando representación de cada apunte (ya multiplicado) pero tenemos que mejorar que apunte terminamos actualizando al guardar. tmb que hacer al agregar nuevo línea de impuesto porque esta deberia multiplicarse en la cantida  de apuntes analiticos correspondientes

jjscarafia committed:
Revert "[FIX] account_invoice_tax: Fixed amount in cc for taxes"

This reverts commit c4cb752af85169a8892da5460cbeafbdeb81442c.

jjscarafia committed:
Revert "[IMP] account_invoice_tax: add company currency to tax wizard"

This reverts commit 79fc435a2413b5ac73c7623599e49aa5747c275b.

jjscarafia committed:
[IMP] _invoice_tax: simplify code

El write de los amounts puede ser más simple, según probamos funciona en todos los casos

jjscarafia committed:
[IMP] _invoice_tax: better ux + protect analytic taxes

1. Mostramos símbolo de la moneda (usamos field monetary)
2. Mejoramos para que en notas de crédito se vea importe positivo si impuesto es postivo
3. Agregamos protección para que no se usen impuestos con setup de analitico porque no está bien implementado y, según analizamos:
* tiene poco uso: solo primicia, gestionen, linos y londero
* no tiene tanto sentido contable
* es complejo de resolver, habŕia que ver si seguimos mostrando representación de cada apunte (ya multiplicado) pero tenemos que mejorar que apunte terminamos actualizando al guardar. tmb que hacer al agregar nuevo línea de impuesto porque esta deberia multiplicarse en la cantida  de apuntes analiticos correspondientes

roboadhoc commented:
[![Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/account-invoicing/pull/222.png)](https://mergebot.adhoc.ar/ingadhoc/account-invoicing/pull/222)

jjscarafia committed:
[IMP] _invoice_tax: branches compatibility + prevent duplicate taxes

1. mejoramos para que funcione bien en branches
2. evitamos que se pueda agregar dos veces mismo impuesto (eso es mejor ux y además terminaba utilizando solo el último)

rov-adhoc commented:
@roboadhoc r+

roboadhoc commented:
@feg-adhoc @rov-adhoc because this PR has multiple commits, I need to know how to merge it:

* `merge` to merge directly, using the PR as merge commit message
* `rebase-merge` to rebase and merge, using the PR as merge commit message
* `rebase-ff` to rebase and fast-forward

rov-adhoc commented:
@roboadhoc r+ rebase-ff

roboadhoc commented:
This PR is already reviewed, reviewing it again is useless.

roboadhoc commented:
Merge method set to rebase and fast-forward.
